### PR TITLE
fix(dashboard): fix edit action in content/monitoring widgets

### PR DESCRIPTION
**Pull request by petrjasek**
it's now opening the item in authoring sidebar

SD-3141

### CODE REVIEW

File: client/app/scripts/superdesk-desks/desks.js [hunk 1]
```diff
@@ -101,8 +101,7 @@
                 };
 
                 scope.edit = function(item) {
-                    desks.setWorkspace(item.task.desk, item.task.stage);
-                    superdesk.intent('author', 'article', item);
+                    superdesk.intent('edit', 'item', item);
                 };
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/desks.js]: Maybe just drop a short docstring here, but it's minor...

**Comment by plamut** [comment]
Very good :+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**